prepare.yml

```diff
@@ -372,7 +372,7 @@
     - name: unarchive Selenium Chrome driver
       become_user: "{{ ansible_user }}"
       tags: selenium-chrome
-      shell: unzip {{ selenium_chrome_download_path }} -o -d {{ selenium_drivers_path }}
+      shell: unzip -o {{ selenium_chrome_download_path }} -d {{ selenium_drivers_path }}
```

Comment:
This now will likely conflict with PR #29, which I just merged. You should probably remove this change from this PR to avoid having to manually fix a conflict in the merge.